doc/command_ref.rst

```diff
@@ -568,48 +581,41 @@ Deplist command
 .. _distro_sync_command-label:
 
 -------------------
-Distro-Sync command
+Distro-Sync Command
 -------------------
 
+| Aliases: ``distro-sync, dsync``
+| Deprecated aliases: ``distrosync, distribution-synchronization``
+
 ``dnf distro-sync [<package-spec>...]``
     As necessary upgrades, downgrades or keeps selected installed packages to match
     the latest version available from any enabled repository. If no package is given, all installed packages are considered.
 
     See also :ref:`\configuration_files_replacement_policy-label`.
 
-------------------------------------
-Distribution-Synchronization command
-------------------------------------
-
-``dnf distribution-synchronization``
-    Deprecated alias for the :ref:`\distro_sync_command-label`.
-
 .. _downgrade_command-label:
 
 -----------------
 Downgrade Command
 -----------------
 
+Aliases: ``downgrade``, ``dg``
+
 ``dnf [options] downgrade <package-spec>...``
     Downgrades the specified packages to the highest installable package of all known lower versions
     if possible. When version is given and is lower than version of installed package then it
     downgrades to target version.
 
-.. _erase_command-label:
-
--------------
-Erase Command
--------------
-
-``dnf [options] erase <spec>...``
-    Deprecated alias for the :ref:`\remove_command-label`.
-
 .. _group_command-label:
 
 -------------
 Group Command
 -------------
 
+| Aliases: ``group``, ``grp``
+| Direct command alises: ``grouplist``, ``groupinstall``, ``groupupdate``, ``groupremove``, ``groupinfo``
```

Comment:
I wonder about the usefulness of these "direct command aliases", they make for an inconsistent interface. Do we really want to keep them or deprecate them as well? Probably a question for @j-mracek

Reply:
I consider them mere `yum` compatibility features. They are not even short-cuts and I would vote for deprecating them.

Reply:
+1, I added them to the deprecated aliases